### PR TITLE
Wording for 2801.

### PR DESCRIPTION
**Pull request by HowardHinnant**
Per request LWG Telecon 2016-12-16.

### CODE REVIEW

File: xml/issue2801.xml [hunk 1]
```diff
@@ -26,9 +26,139 @@ satisfied.
...
+<pre>
+constexpr unique_ptr() noexcept;
+</pre>
+<blockquote>
+<p>...</p>
+<p>
+4 <i>Remarks:</i> <del>If this constructor is instantiated with a
+pointer type or reference type for the template argument <tt>D</tt>,
+the program is ill-formed.</del> <ins>If <tt>deleter_type</tt> is a
+pointer type or is not <tt>DefaultConstructible</tt> (Table 22), this
```
Comment (timsong-cpp) [inline xml/issue2801.xml]: This should say "`is_default_constructible_v<deleter_type>` is `false`".

File: xml/issue2801.xml [hunk 2]
```diff
@@ -26,9 +26,139 @@ satisfied.
...
+<pre>
+explicit unique_ptr(pointer p) noexcept;
+</pre>
+<blockquote>
+<p>...</p>
+<p>
+8 <i>Remarks:</i> <del>If this constructor is instantiated with a
+pointer type or reference type for the template argument <tt>D</tt>,
+the program is ill-formed.</del> <ins>If <tt>deleter_type</tt> is a
+pointer type or is not <tt>DefaultConstructible</tt> (Table 22), this
```
Comment (timsong-cpp) [inline xml/issue2801.xml]: ...and here too.

File: xml/issue2801.xml [hunk 3]
```diff
@@ -26,9 +26,139 @@ satisfied.
...
+<ins><i>Requires:</i> <tt>deleter_type</tt> shall satisfy the requirements of
+<tt>DefaultConstructible</tt> (Table 22), and that construction shall not throw
+an exception.</ins>
+</p>
+<p>
+<ins><i>Effects:</i> Equivalent to the <tt>unique_ptr</tt> default constructor.</ins>
+</p>
+<p>
+<ins><i>Remarks:</i> If <tt>deleter_type</tt> is a pointer type or is
+not <tt>DefaultConstructible</tt> (Table 22), this constructor shall
```
Comment (timsong-cpp) [inline xml/issue2801.xml]: ...and here too.

File: xml/issue2801.xml [hunk 4]
```diff
@@ -26,9 +26,139 @@ satisfied.
...
+constructor shall not participate in overload resolution.</ins>
+</p>
+</blockquote>
+
+<pre>
+<ins>constexpr unique_ptr(nullptr_t) noexcept;</ins>
+</pre>
+<blockquote>
+<p>
+<ins><i>Requires:</i> <tt>deleter_type</tt> shall satisfy the requirements of
```
Comment (timsong-cpp) [inline xml/issue2801.xml]: I think this paragraph is implied by the "equivalent to" magic words, but doesn't hurt.

File: xml/issue2801.xml (hunk 1 above)
Comment (HowardHinnant) [inline xml/issue2801.xml]: Thanks, agreed, fixed.

**Pull request merged**